README.md

```diff
@@ -20,16 +20,16 @@ k3sup was developed to automate what can be a very manual and confusing process
 
 ### Uses
 
-* Bootstrap Kubernetes with k3s onto any VM - either manually, during CI or through `cloudinit`
-* Get from zero to `kubectl` with `k3s` on Raspberry Pi (RPi), VMs, AWS EC2, Packet bare-metal, DigitalOcean, Civo, Scaleway, and others
-* Fetch a working KUBECONFIG from an existing `k3s` cluster
-* Join nodes into an existing `k3s` cluster with `k3sup join`
-* Install selected helm charts without `tiller` with `k3sup app install`
+- Bootstrap Kubernetes with k3s onto any VM - either manually, during CI or through `cloudinit`
```

Comment:
Curious why you changed this?

Reply:
Same answer on both, vscode plugin: davidanson.vscode-markdownlint with good practices and rules on .md files, I don't mind to change it back as it was without the linting 👌